spec/models/web_hook_events_daily_aggregate_spec.rb

```diff
@@ -0,0 +1,84 @@
+# frozen_string_literal: true
+
+RSpec.describe WebHookEventsDailyAggregate do
+  let!(:web_hook) { Fabricate(:web_hook) }
+  let!(:event) do
+    WebHookEvent.create!(status: 200, web_hook: web_hook, created_at: 1.days.ago, duration: 280)
+  end
+  let!(:event_today) { WebHookEvent.create!(status: 200, web_hook: web_hook, duration: 300) }
```

Comment:
Can you use a `fab!` instead for these?

```ruby
fab!(:event_today) { Fabricate(:web_hook_event, status: 200, web_hook: web_hook, duration: 300) }
```

This way you don't have to use `event_today` for the creation to happen for example. They'll all be created before the tests run.
```